.travis.yml

```diff
@@ -4,3 +4,19 @@ node_js:
   - '0.10'
   - '0.12'
   - 'iojs'
+
+cache:
+  directories:
+    - node_modules
+
+matrix:
+  fast_finish: true
```

Comment:
`fast_finish` permit to mark build at fail as soon as one env failed (node 0.10, node 0.12, io.js)